ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -599,9 +599,59 @@
           started_status: null
           wrappers: []
           publishers: []
+      - '{name}-{test_name}-no-scm':
+          test_name: windows-e2e-proxyall-pending-label
+          node: null
+          description: 'This is for marking PR as pending for e2e proxyall test.'
+          builders:
+            - builder-pending-label
+          trigger_phrase: null
+          white_list_target_branches: []
+          allow_whitelist_orgs_as_admins: false
+          admin_list: []
+          org_list: []
+          white_list: []
+          only_trigger_phrase: false
+          trigger_permit_all: true
+          throttle_concurrent_builds_category:
+          throttle_concurrent_builds_enabled: 'false'
+          status_context: jenkins-windows-proxyall-e2e
+          status_url: --none--
+          success_status: Pending test. Mark as failure. Add comment /test-windows-proxyall-e2e to trigger.
```

Comment:
For the Windows e2e job, we only need to keep windows-e2e and windows-e2e-userspace.